scripts/testNetworkLiquidAssets.ts

```diff
@@ -0,0 +1,157 @@
+// Copyright 2023 Parity Technologies (UK) Ltd.
+
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import { Keyring } from '@polkadot/keyring';
+import { KeyringPair } from '@polkadot/keyring/types';
+import { cryptoWaitReady } from '@polkadot/util-crypto';
+import chalk from 'chalk';
+
+import { KUSAMA_ASSET_HUB_WS_URL } from './consts';
+import { awaitBlockProduction, delay, logWithDate } from './util';
+
+const ASSET_ID = 1;
+const ASSET_NAME = 'Testy';
+const ASSET_TICKER = 'TSTY';
+const ASSET_DECIMALS = 12;
+const ASSET_MIN = 1;
+
+const asset = {
+	parents: 0,
+	interior: {
+		X2: [{ palletInstance: 50 }, { generalIndex: ASSET_ID }],
+	},
+};
+
+const native = {
+	parents: 1,
+	interior: {
+		Here: '',
+	},
+};
+
+const createAssetCall = (api: ApiPromise, admin: KeyringPair) => {
+	return api.tx.assets.create(ASSET_ID, admin.address, ASSET_MIN);
+};
+
+const setMetadataCall = (api: ApiPromise) => {
+	return api.tx.assets.setMetadata(ASSET_ID, ASSET_NAME, ASSET_TICKER, ASSET_DECIMALS);
+};
+
+const mintCall = (api: ApiPromise, to: KeyringPair, amount: number) => {
+	return api.tx.assets.mint(ASSET_ID, to.address, amount);
+};
+
+const createLiquidityPoolCall = (api: ApiPromise) => {
+	// For now, we have to override the types of the Assets until PJS is updated
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+	return api.tx.assetConversion.createPool(<any>native, <any>asset);
```

Comment:
This is totally fine with me. I double checked the ts compilter issue happening with PJs and this is the right approach.